docs/sources/operators-guide/deploy-grafana-mimir/upgrade-helm-chart-2.1-to-3.0/_index.md

```diff
@@ -125,6 +127,13 @@ Several parameters that were available in version 2.1 of the mimir-distributed H
        replicas: 3
      ```
 
+1. Decide whether or not you need to update the Mimir configuration due to memcached changes:
```

Comment:
There is already a section about the mimir.config which mentions memcached. It's called "Update the `mimir.config` value, based on the following information:" Do you think you can move this there?